client/web/src/nav/Feedback/FeedbackPrompt.tsx

```diff
@@ -165,18 +167,21 @@ export const FeedbackPrompt: React.FunctionComponent<Props> = ({ open, history,
     const handleToggle = useCallback(() => setIsOpen(open => !open), [])
     const forceClose = useCallback(() => setIsOpen(false), [])
     const match = useRoutesMatch(routes)
+    const [isRedesignEnabled] = useRedesignToggle()
 
     return (
         <ButtonDropdown a11y={false} isOpen={isOpen} toggle={handleToggle} className="feedback-prompt" group={false}>
             <DropdownToggle
                 tag="button"
                 caret={false}
-                className="btn btn-link btn-sm text-decoration-none feedback-prompt__toggle"
-                nav={true}
+                className={classNames('btn btn-sm text-decoration-none feedback-prompt__toggle', {
+                    'btn-secondary-outline': isRedesignEnabled,
```

Comment:
It seems it should be `btn-outline-secondary` to apply Tom's updated styles.

```suggestion
                    'btn-outline-secondary': isRedesignEnabled,
```